src/BreakfastSerializer/Plugin/IsPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializer\Plugin;
+
+/**
+ * Interface IsPlugin
+ * @package BDBStudios\BreakfastSerializer\Plugin
+ */
+interface IsPlugin
```

Comment:
PSR standard the interface should be appended with `Interface` https://github.com/php-fig/fig-standards/blob/master/bylaws/002-psr-naming-conventions.md